distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc

```diff
@@ -0,0 +1,708 @@
+:title: NSILI (STANAG 4559) Attribute Mappings
+:type: subAppendix
+:order: 900
+:parent: Format-specific Attribute Mappings
+:status: published
+:summary: NSILI (STANAG 4559) Attribute Mappings.
+
+== {title}
+
+.NSILI (STANAG 4559) Attribute Mappings
+[cols="5" options="header"]
+|===
+
+|NSILI Entity Field
+|NSILI Attribute Field
+|Term
+|Translation
+|Type
+
+|NSIL_ALL_VIEW
+4+|_no direct fields_
+
+|NSIL_APPROVAL
+|approvedBy
+3+|_not mapped_
+
+|NSIL_APPROVAL
+|dateTimeModified
+3+|_not mapped_
+
+|NSIL_APPROVAL
+|status
+3+|_not mapped_
+
+|NSIL_ASSOCIATION
+4+|_no direct fields_
+
+|NSIL_ASSOCIATION_VIEW
+4+|_no direct fields_
+
+|NSIL_CARD
+|dateTimeModified
+|`modified`
+|none
+|Date
+
+|NSIL_CARD
+|identifier
+|add a value to `metacard.associations.related` if the identifier refers to an associated metacard (i.e. the card is an association node); +
+else, `id`
+|none
+|String
+
+|NSIL_CARD
+|numberOfParts
+3+|_not mapped_
+
+|NSIL_CARD
+|publisher
+|`contact.publisher-name`
+|none
+|String
+
+|NSIL_CARD
+|sourceDateTimeModified
+|`created`
+|none
+|Date
+
+|NSIL_CARD
+|sourceLibrary
+3+|_not mapped_
+
+|NSIL_CARD
+|status
+3+|_not mapped_
+
+|NSIL_CBRN
+|alarmClassification
+3+|_not mapped_
+
+|NSIL_CBRN
+|cbrnCategory
+3+|_not mapped_
+
+|NSIL_CBRN
+|eventType
+3+|_not mapped_
+
+|NSIL_CBRN
+|incidentNumber
+3+|_not mapped_
+
+|NSIL_CBRN
+|operationName
+3+|_not mapped_
+
+|NSIL_CBRN
+|substance
+3+|_not mapped_
+
+|NSIL_CBRN_VIEW
```

Comment:
Views are just a subset of the Nodes that apply to certain types of data (e.g. CBRN for this one), so they can be omitted from the table